docs/reference/pvc-finalizer-downgrade-issue.md

```diff
@@ -0,0 +1,32 @@
+* TOC
+{:toc}
+
+---
+title: Kubernetes Downgrade issue from 1.10 to 1.9 due to PV/PVC Protection
+---
+
+## PV/PVC Protection in Kubernetes 1.10
+
+When enabled, [PV/PVC Protection](docs/admin/admission-controllers.md#storage-object-in-use-protection-beta) prevents PV/PVC from being removed when the finalizers are removed. 
+
+## Downgrading issue
+
+After downgrading from Kubernetes 1.10 to 1.9, PV/PVCs that are created in Kubernetes 1.10 with PVC Protection cannot be removed.
+
+## Workaround
+
+Currently PV/PVC finalizers have to be manually removed so PV/PVC can be removed after downgrading to Kubernetes 1.9.
+
+Before downgrading to Kubernetes 1.9, disable `StorageObjectInUseProtection` plugin and restart admission controller.
```

Comment:
"1. Before you downgrade, disable the `StorageObjectInUseProtection` plugin and restart the admission controller."

Can we also please provide the shell command here? (and if you indent four spaces it will keep the step numbering, with my great thanks)

Reply:
`hyperkube apiserver --disable-admission-plugins=StorageObjectInUseProtection ... (rest of the options omitted)`

Reply:
Or `kube-apiserver --disableadmissionplugins=StorageObjectInUseProtection ...`? (to match what's currently in the docs)

Also -- I see nothing in the admission controller docs about a restart. If it's necessary, what's the command? Do you have to restart the apiserver?

Reply:
`--disable-admission-plugins` is the command line option, there is `-` in between words. 

The restart process is to stop apiserver, and start it again but with a different command line option as above:
- if apiserver is managed by `systemd`, stop it using `systemd stop <apiserver service name>`. Modify apiserver service unit service file, reload the service, and start it using `systemd start` command
- if apiserver is started through command line, kill the process and run the command again with the above command line option